examples/todos/app.js

```diff
@@ -1,19 +1,38 @@
-
-/**
- * Module dependencies.
- */
-
-var express = require('express')
-  , path = require('path')
-  , morgan = require('morgan');
-
+var http = require('axios');
+var read = require('read-yaml');
+var PouchDB = require('pouchdb');
+var express = require('express');
+var path = require('path');
 var app = express();
+var config = read.sync('./config.yml');
 
+const DB_URL = `${config.protocol}${config.domain}:${config.port}${config.dbServerEndpoint}`
+const DB_ADMIN_URL = `${config.protocol}${config.admin.username}:${config.admin.password}@${config.domain}:${config.port}${config.dbServerEndpoint}`
+
+app.use(config.dbServerEndpoint, require('express-pouchdb')(PouchDB.defaults({prefix: './db/'})));
+app.listen(config.port);
 
-app.use(morgan('dev'));
 app.use(express.static(path.join(__dirname, 'public')));
-app.use('/db', require('../../'));
 
+async function setup() {
+  
+  // Set up the admin user.
+  try {
+    await http.put(`${DB_URL}/_config/admins/${config.admin.username}`, `"${config.admin.password}"`, {headers:{}});
+    console.log("Admin created.");
+  }
+  catch (err) {
+    console.log("We already have admins."); 
+  }
+
+  // Set up the app database.
```

Comment:
Similarly, if you store the ``PouchDB.defaults()`` intermediate result above, you could just do ``new PouchDB('todos')`` here. Again, it removes the HTTP roundtrip.